wrapper/src/test/java/software/amazon/jdbc/plugin/efm/ConcurrencyTests.java

```diff
@@ -62,6 +62,11 @@
 import software.amazon.jdbc.dialect.Dialect;
 import software.amazon.jdbc.dialect.UnknownDialect;
 
+import static software.amazon.jdbc.plugin.efm.HostMonitoringConnectionPlugin.FAILURE_DETECTION_COUNT;
+import static software.amazon.jdbc.plugin.efm.HostMonitoringConnectionPlugin.FAILURE_DETECTION_INTERVAL;
+import static software.amazon.jdbc.plugin.efm.HostMonitoringConnectionPlugin.FAILURE_DETECTION_TIME;
+import static software.amazon.jdbc.plugin.efm.MonitorServiceImpl.MONITOR_DISPOSAL_TIME_MS;
```

Comment:
Looks like the `Main CI / Run non-container integration tests` check is failing. Checkstyle is complaining about these static imports being in the wrong order. I believe they should be above the non static imports.